llvm/test/CodeGen/AArch64/intrinsic-cttz-elts-sve.ll

```diff
@@ -156,53 +137,178 @@ define i64 @vscale_4096_poison(<vscale x 16 x i8> %a) #1 {
   ret i64 %res
 }
 
-; NO VSCALE RANGE
+; MATCH WITH BRKB + CNTP
```

Comment:
nit: This isn't quite true as it might match with BRKB + INCP, although I understand what you mean. Perhaps you could rewrite this as:

`; EFFICIENT LOWERING USING BRKB`

?

Reply:
Good point - Done.